core/trino-main/src/main/java/io/trino/execution/SetSessionTask.java

```diff
@@ -110,7 +110,7 @@ public ListenableFuture<Void> execute(
             propertyMetadata.decode(objectValue);
         }
         catch (RuntimeException e) {
-            throw semanticException(INVALID_SESSION_PROPERTY, statement, e.getMessage());
+            throw semanticException(INVALID_SESSION_PROPERTY, statement, "Invalid session property value '%s': %s", propertyName.toString(), e.getMessage());
```

Comment:
SET SESSION is invoked for a particular session property.
we don't need to provide the property name in the exception message, do we?

Reply:
awesome. can this be a separate change, or even separate PR?
this change is orthognal to error-prone